applications/DEMApplication/custom_elements/analytic_spheric_particle.h

```diff
@@ -170,7 +170,6 @@ void EvaluateBallToBallForcesForPositiveIndentiations(SphericParticle::ParticleD
 void ComputeBallToRigidFaceContactForce(SphericParticle::ParticleDataBuffer & data_buffer,
                                         array_1d<double, 3>& r_elastic_force,
                                         array_1d<double, 3>& r_contact_force,
-                                        double& RollingResistance,
```

Comment:
`r_rolling_resistance` should be the style according to what I see in DEMApp. Don you have any style convention in DEM @maceligueta ?

Reply:
no problem, we are close to removing it forever.